teachbooks/cli/main.py

```diff
@@ -45,9 +46,31 @@ def build(ctx, path_source, publish, process_only):
             toc=path_toc
         )
 
+@main.group(invoke_without_command=True)
+@click.pass_context
+def serve(ctx):
+    """Start a web server to interact with the book locally"""
+    from teachbooks.serve import Server
+    
+    if ctx.invoked_subcommand is None:
+        # Hardcoded for now
+        dir = Path("./book/_build/html/")
+        workdir = Path("./book/.teachbooks")
+        server = Server(dir=dir, workdir=workdir)
+
+        server.start()
+        echo_info(f"server running on {server.url}")
+
+
+@serve.command()
+def stop():
+    """Stop the webserver"""
+    from teachbooks.serve import Server
+    server = Server.load(Path("./book/.teachbooks"))
+    server.stop()
```

Comment:
can `teachbooks/server/state.pickle` be moved to the root? This avoids problems if users deviate from our convention where `book` contains all source code for the book. Match convention for subdirectory, too, so if in `book`, use `_teachbooks` and if in root looks like `.teachbooks` is best (matches CI, github, `.venv` ent).

The pickle should be ignored anyway, right? Will it be cleaned up after server closes?

Reply:
I'll move `.teachbooks` to the root directory, I think it indeed makes more sense to put it there. I use `.teachbooks` and not `_teachbooks`, because the dot makes the folder hidden on linux (user shouldn't have to interact with the folder ever anyway).

Yeah pickle should be deleted after stop, but I see that that's not implemented yet. Good catch :)